package.json

```diff
@@ -22,8 +22,9 @@
   },
   "jest": {
     "coveragePathIgnorePatterns": [
-      "src/server/database"
-    ]
+      "<rootDir>/src/server/database"
+    ],
+    "globalSetup": "<rootDir>/src/tests/globalSetup.js"
```

Comment:
This file is used to prepare stuff before we actually run tests. Here we simply run `dotenv.config()` to have the proper env variables.